.bingo/operator-sdk.mod

```diff
@@ -1,11 +1,13 @@
 module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
-go 1.23.4
+go 1.24.3
+
+toolchain go1.24.6
 
 replace github.com/containerd/containerd => github.com/containerd/containerd v1.4.11
 
 replace github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
 
 replace github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
 
-require github.com/operator-framework/operator-sdk v1.39.1 // cmd/operator-sdk -ldflags=-X=github.com/operator-framework/operator-sdk/internal/version.Version=v1.34.1
+require github.com/operator-framework/operator-sdk v1.41.1 // cmd/operator-sdk -tags=containers_image_openpgp -ldflags=-X=github.com/operator-framework/operator-sdk/internal/version.Version=v1.41.1
```

Comment:
-tags=containers_image_openpgp

Interesting, it be added. 

This one I did not updated in that PR because it was not working.
So, probably this flag sorted it out.